hack/verify-flags/known-flags.txt

```diff
@@ -47,6 +47,7 @@ cluster-name
 cluster-tag
 concurrent-deployment-syncs
 concurrent-endpoint-syncs
+concurrent-replicaset-syncs
```

Comment:
This doesn't match the flag name in pkg/controller/replicaset/options/options.go